### ISSUE TITLE
Services page - Rething the design?

### ISSUE THREAD

**Issue by uumoss-bb** [uumoss-bb/NOX-2.0#54]
I wanna know what you guys think of the services display. the grey squares. I know I design them but I think there is a better option here.

**Comment by dmoyer21** [comment]
I think the grey squares are fine. I just don't like the size of the text box with this one. It should be the same size as the grey box, and we need to cut down the wording enough to the point that you don't have to scroll at all to read the service. I believe people like simplicity, which means least amount of reading as possible. 

Another idea would be for us to create a picture with dialogue on it explaining the services, with buttons at the bottom for people who are interested. 

Are we keeping the testimonials with the services page. Or making that a separate tab?

**Comment by uumoss-bb** [comment]
I agree with trying to keep it simplistic, that on you guys to reword it. We will be working on a way to make the text box do bullets. 

I think it would be sweet to throw some pictures up.

I don't see a reason to separate the testimonial from services. I think they kind of go hand in hand.
